enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2054,6 +2081,44 @@ func (ws *workspace) fetch(ctx context.Context, remoteURL string, refs []string,
 	return nil
 }
 
+// Writes a wrapper script that invokes the ci_runner with the bazel_wrapper subcommand.
+// Also adds it to the PATH so it will be invoked whenever `bazel` or `bazelisk` are called.
+// The wrapper script adds a startup option for the custom ci_runner .bazelrc to
+// all bazel commands.
+func (ws *workspace) writeBazelWrapperScript() error {
+	wrapperDir := filepath.Join(ws.rootDir, "wrappers")
+	for _, c := range []string{bazelBinaryName, bazeliskBinaryName} {
+		wrapperPath := filepath.Join(wrapperDir, c)
+		_, err := os.Stat(wrapperPath)
+		if err != nil {
+			if os.IsNotExist(err) {
+				if err := os.MkdirAll(filepath.Dir(wrapperPath), 0755); err != nil {
+					return err
+				}
+			} else {
+				return err
+			}
+		}
+
+		contents := `
+#!/usr/bin/env sh
```

Comment:
hm I guess this probably works because the shell is the default executable interpreter? but usually the `#!` has to be on the first line

```suggestion
		contents := `#!/usr/bin/env sh
```

Reply:
Done